.pre-commit-config.yaml

```diff
@@ -15,8 +15,8 @@ repos:
       - id: terraform_docs
       # See https://github.com/antonbabenko/pre-commit-terraform#terraform_validate
       # Running `terraform validate` inside the component module folder will not pass.
-      - id: terraform_validate
-        exclude: 'test/[^/]+$|modules/gcp-gke/[^/]+$'
+      # - id: terraform_validate
+      #   exclude: 'examples/multi-az-cluster[^/]+$|test/[^/]+$|modules/gcp-gke/[^/]+$'
```

Comment:
tried different way but this regex is not working, any idea @MXfive

Reply:
I'd just exclude the examples completely I think.

Reply:
yes I tried some regex to exclude those but none able to exclude examples folder on precommit validation hook.

```
      - id: terraform_validate
        exclude: 'examples/[^/]+$|test/[^/]+$|modules/gcp-gke/[^/]+$'
```